src/http/response.rs

```diff
@@ -45,6 +51,43 @@ impl Response<Bytes> {
         Ok(resp)
     }
 
+    pub fn get_header_value(&self, header_name: &str) -> Option<String> {
+        self.headers
+            .get(header_name)
+            .and_then(|header_value| header_value.to_str().ok())
+            .map(|s| s.to_string())
+    }
+
+    pub fn to_grpc_error(
+        self,
+        status_details: &Option<ProtobufMessage>,
+    ) -> Result<Response<async_graphql::Value>> {
```

Comment:
this returns error anyway, right?

probably, need just return an error instead of Result

Reply:
Yes, agree. I'll fix fix this